### PR TITLE
model association changes

### PR DESCRIPTION
**Pull request by SauravGaur1**

### CODE REVIEW

File: src/api/v1/Certification/model.js
Comment (rks369) [inline src/api/v1/Certification/model.js:1]: key is missing

File: src/api/v1/City/model.js
Comment (rks369) [inline src/api/v1/City/model.js:1]: key is missing

File: src/api/v1/Certification/model.js
Comment (SauravGaur1) [reply src/api/v1/Certification/model.js:1]: @rks369 kindly check and resolve

**Review by SauravGaur1 — COMMENTED** [review]
Approved

**Comment by SauravGaur1** [comment]
Models associations

**Pull request merged**